api/src/main/java/org/openmrs/util/PrivilegeConstants.java

```diff
@@ -880,4 +880,14 @@
 	
 	@AddOnStartup(description = "Able to manage the search index")
 	public static final String MANAGE_SEARCH_INDEX = "Manage Search Index";
+	
+	@AddOnStartup(description = "Able to add an order set")
+	public static final String ADD_ORDERSET = "Add OrderSet";
```

Comment:
Should have a single "Manage Order Sets" privilege instead of separate ones for Add and Edit. (This is the OpenMRS convention for _Metadata_ (whereas _Data_ does have separate Add and Edit privs).